data_preprocessing/k_neighbors_classifier.py

```diff
@@ -0,0 +1,23 @@
+from classifier import Classifier
+from sklearn.neighbors import KNeighborsClassifier
+from sklearn.metrics import accuracy_score, confusion_matrix, classification_report
+
+
+class KNeighborsClassifierModel(Classifier):
+    def train_and_evaluate(self):
+        X_train = self.training_data.drop('Class', axis=1)
+        y_train = self.training_data['Class']
+        X_test = self.test_data.drop('Class', axis=1)
+        y_test = self.test_data['Class']
+
+        model = KNeighborsClassifier()
```

Comment:
❓Do you guys have any example of testing result? I see that sklearn's classifiers have extra params and I wonder if we can't use them for better results